### DISCUSSION TITLE
Change to `specie`

### DISCUSSION THREAD

**Discussion by zerothi** [zerothi/sisl#732] [Ideas]
Hi @tfrederiksen can we change `sa` to `specie`, isn't this more declarative?

**Comment by tfrederiksen** [comment]
What about `species_*` (with s)? I think this is the most conventional form both in singular and plural forms: https://www.merriam-webster.com/dictionary/species.

In `io/siesta` I find `species_Z`. Then maybe we should use `species_label` when the identifier is a string?

**Reply by zerothi** [reply]
Ok, I agree `species` would be fine. I would also like to change it in siesta to only `species`. wouldn't this be ok?

The species, I guess, can change depending on names in the input file.

**Reply by zerothi** [reply]
I'll do it ;)

**Reply by tfrederiksen** [reply]
Sounds good to me.

**Reply by zerothi** [reply]
Done!

**Reply by zerothi** [reply]
In the end I didn't go for the species_label. If we should do something we should cast it to an `Atom` but that might also be a bit overkill? Or?
